content/api/editor-support.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Editor support
+---
+
+Installing our code locally will allow your editor to extract information from our code, giving you additional features like auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
+
+Installing these definitions is optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
+
+## Installation
+Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. 
+
+```bash
+pip install git+https://github.com/sourcebots/robot-api
+```
+And on windows:
+
+```powershell
+py -m pip install git+https://github.com/sourcebots/robot-api
+```
+
+If you're doing this on a shared computer, such as those found in your colleges, add `--user` to the end:
```

Comment:
This is making an assumption about the setup in the colleges. I suggest changing to: "such as those you may find in your college"